kubernetes-deployment/test/test_maintainlib_common.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import unittest
+import filecmp
+import os
+import yaml
+import tarfile
+import shutil
+import sys
+
+from maintainlib import common
+
+
+class TestMaintainlibCommon(unittest.TestCase):
+
+    """
+    Test the common's api
+    """
+
+    def setUp(self):
+
+        try:
+
+            os.chdir(os.path.abspath("test"))
+
+        except:
+
+            pass
+
+
+
+
+    def tearDown(self):
+
+        try:
+
+            os.chdir(os.path.abspath(".."))
+
+        except:
+
+            pass
+
+
+
+    def test_yaml_lod(self):
```

Comment:
>test_yaml_lod [](start = 8, length = 13)

test_yaml_load? maybe you can add a comment  "todo"?

Reply:
Maybe we can remove this function. Because whether a yaml file load correctly or not depends on the lib.